src/z3c/autoinclude/utils.py

```diff
@@ -43,6 +43,15 @@ def __init__(self, zcml_to_look_for):
         for zcml_group in zcml_to_look_for:
             self[zcml_group] = []
 
+    def __bool__(self):
+        for value in self.values():
+            if value:
+                return True
+        return False
```

Comment:
maybe `return any(self.values())` is equivalent and more concise.

Reply:
Ah, good point. Done.